### ISSUE TITLE
Balance out card sizes between main deck & sideboard

### ISSUE THREAD

**Issue by tooomm** [Senryoku/Draftmancer#543]
When the sideboard is minimized and displayed next to the main deck, cards in sideboard are of fixed size and kinda big. (I'm using a smaller screen here to show it in an extremer form):
<img width="772" alt="image" src="https://github.com/Senryoku/Draftmancer/assets/9874850/4e380084-1437-48ea-82b6-84e1c3243d46">
<img width="770" alt="image" src="https://github.com/Senryoku/Draftmancer/assets/9874850/c1c8319b-0514-440d-b101-b1a420b33cad">

Ideally, all cards displayed would always share the same size.

**Comment by Senryoku** [comment]
Yeah, I'm aware of this issue on smaller screen, I didn't really want to touch it for a stupid reason: The title size is tuned for this exact size of cards. I'll try adjusting the font size when the cards are too small (or just allow wrapping).

I'm afraid making all columns of the same size will be pretty tricky as it could lead to non converging (or at least long) feedback loops depending on how I implement it, but it might be worth a try.
Do you think making the sideboard resizable could be a better solution?
